frontend/ui/lib/widgets/place_block_widget.dart

```diff
@@ -13,17 +13,40 @@ class _PlaceBlockWidgetState extends State<PlaceBlockWidget> {
   final String placeName;
   _PlaceBlockWidgetState(this.placeName);
 
+  List<bool> _selections = List.generate(3, (_) => false);
+
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-          border: Border.all(
-            color: Colors.black,
-          ),
-          borderRadius: BorderRadius.all(Radius.circular(5))),
-      margin: const EdgeInsets.all(15.0),
-      padding: const EdgeInsets.all(10.0),
-      child: Text(placeName),
-    );
+    return ExpansionTile(
+        title: Text(placeName),
+        trailing: ToggleButtons(
+          children: <Widget>[
+            Text('M'),
```

Comment:
The meaning/behavior of "M" and "T" is unclear here. We should either add some external explanation somewhere as to what this is or add a tooltip to each button (which displays text on hover). This also seems like a great place to take advantage of the [Material Design](https://material.io/resources/icons/?style=baseline) icons [available in Flutter](https://api.flutter.dev/flutter/material/Icons-class.html).

Reply:
I like the tooltip and icon ideas

Reply:
Yeah adding a tool tip would definitely help the user understand what's going on I'll add that.